src/shadowbox/Taskfile.yml

```diff
@@ -27,6 +27,9 @@ tasks:
       TARGET_DIR: '{{.TARGET_DIR | default (joinPath .OUTPUT_BASE .TARGET_OS .TARGET_ARCH)}}'
       NODE_DIR: '{{joinPath .TARGET_DIR "app"}}'
       BIN_DIR: '{{joinPath .TARGET_DIR "bin"}}'
+      SB_VERSION: '{{.SB_VERSION}}'
+    env:
+      SB_VERSION: '{{.SB_VERSION}}'
```

Comment:
Which command needs this? I'd favor passing it to the specific command instead, so it's clearer.

Reply:
The `npx webpack ...` line. Will pass it there!